src/components/media-player/media-player-track.js

```diff
@@ -213,7 +213,7 @@ function MediaPlayerTrack({
     lightModeValue = false;
   } else if (posterImageUrl) {
     lightModeValue = posterImageUrl;
-  } else if (youtubeThumbnailUrl.lowResThumbnailUrl) {
+  } else if (youtubeThumbnailUrl?.lowResThumbnailUrl) {
```

Comment:
Tiny thing: makes sense to only check for `youtubeThumbnailUrl` here now, right?

Reply:
Ahh so true, after this change, checking for `youtubeThumbnailUrl` is enough of course. Changed! Thanks!